### DISCUSSION TITLE
Loading Data in Chunks

### DISCUSSION THREAD

**Discussion by BPC-001** [kernc/backtesting.py#619] [Q&A]
Hello - I was wondering if anyone has experimented with loading data in chunks. The use case it that we'd like to run a backtest against 20 years worth of tickframe data, and therefore, were considering loading it in chunks. 

Thanks in advance for any suggestions!

**Comment by samueltg92** [comment]
Hello!
I don't know if it helps, but I use this:

**_60% of the dataset_**

df60 = df[0:270000]  

**_40% of the dataset_**

df40 = df[270000:449192]  
df60.dropna()
df40.dropna()
